_posts/en/newsletters/2019-10-30-newsletter.md

```diff
@@ -19,14 +19,15 @@ the Bitcoin StackExchange.
 ## Action items
 
 - **Upgrade to C-Lightning 0.7.3:** this latest [release][c-lightning
-  0.7.3] adds support for a PostgresSQL backend, makes it possible to
+  0.7.3] adds support for a PostgreSQL backend, makes it possible to
   send funds directly to a particular address when closing a channel,
   and allows you keep your HD wallet seed encrypted when `lightningd`
   isn't running---plus many other features and several bug fixes.
 
-- **Help Bitcoin Core release candidate:** experienced users are encouraged to
+- **Help test Bitcoin Core release candidate:** experienced users are encouraged to
   help test the latest release candidates for the upcoming version of
-  [Bitcoin Core][Bitcoin Core 0.19.0].
+  [Bitcoin Core][Bitcoin Core 0.19.0] and [report][Bitcoin Core #17079]
+  any issues you find.
```

Comment:
I don't know if you want to get into the details, but what is desired afaict is for people to report their RC testing in the meta-issue (to reassure the maintainers that testing is taking place and for which platforms) and to open a separate issue if a problem is encountered while testing.

Reply:
That's confusing.  I'm going to drop mention of this issue.